backend/lib/models/index.js

```diff
@@ -6,7 +6,9 @@ const PATH_TO_SCHEMAS = join(__dirname, "./schemas");
 module.exports = function registerModels(mongoConnection) {
   readdirSync(PATH_TO_SCHEMAS).forEach((fileName) => {
     const filePath = join(PATH_TO_SCHEMAS, fileName);
+    console.log({ fileName, filePath });
```

Comment:
This file is one of the worst in the whole project, using IO to read mongo model for no reason/benefit

Reply:
This is a convenient way to auto-load the mongoose models.
I've used similar methods before to remove some boilerplate and often repeating code, I find it also better to inject the dependencies this way for testing and such.
There is even an official Fastify plugin doing the same for plugins https://github.com/fastify/fastify-autoload

Reply:
Also, this is out of scope of the current Pull Request and was already merged to master.